blocks/library/text/index.js

```diff
@@ -31,13 +31,29 @@ registerBlockType( 'core/text', {
 		content: query( 'p', children() ),
 	},
 
+	transforms: {
+		from: [
+			{
+				type: 'raw',
+				matcher: ( node ) => (
+					node.nodeName === 'P' &&
```

Comment:
Good point, maybe we should consider adding a boolean parameter to the matcher function. These transformations will be really similar